docs/topics/downloader-middleware.rst

```diff
@@ -373,6 +373,7 @@ what is implemented:
 * Revalidate stale responses based on `Last-Modified` response header
 * Revalidate stale responses based on `ETag` response header
 * Set `Date` header for any received response missing it
+* Support `max-stale` cache-control directive in requests
```

Comment:
To use this feature developer must know details of RFC2616 - not many people read this 178-page manuscript. @jameysharp's commit comments were great, e.g. 

> This allows spiders to be configured with the full RFC2616 cache policy,
but avoid revalidation on a request-by-request basis, while remaining
conformant with the HTTP spec.

I think we shouldn't loose these comments. Something like "Add `Cache-Control: max-stale=600` to Request headers to ..."

See also: RFC2616, 14.9.3